flow/client_server/static/internal/groups/create/js/script.js

```diff
@@ -0,0 +1,36 @@
+function onceDocumentReady() {
+	getURLs(startSubmitButtonListener);
+}
+
+function startSubmitButtonListener() {
+	$('#submit').click(function () {
+		var group_name = $('#group_name').val();
+		var aid = getAID();
+		var data = {
+			group_name: group_name,
+			aid: aid
+		};
+
+		$.ajax({
```

Comment:
It's an event handler for when the element is clicked. Useful for a like button for example.

Read more about the [.click function here](https://api.jquery.com/click/)